code/modules/mob/mob_movement_old.dm

```diff
@@ -0,0 +1,529 @@
+/mob/CanPass(atom/movable/mover, turf/target, height=0, air_group=0)
```

Comment:
Is this whole file actually used for anything?

Reply:
it is not anymore, i just had it as a reference for integrating the new system to existing code, i will remove it